jenkins/job-configs/kubernetes-jenkins/clean-ci-node-e2e.yaml

```diff
@@ -0,0 +1,18 @@
+- job:
+    name: 'clean-ci-node-e2e'
+    description: 'Clean up leaked node e2e resources. Test owner: test-infra maintainers.'
+    node: ''
```

Comment:
I think this needs to be `master` or it won't get scheduled anywhere on CI jenkins.